packages/backend/src/routers/apiV1Router.ts

```diff
@@ -52,6 +53,10 @@ apiV1Router.get('/flash', (req, res) => {
 
 apiV1Router.post('/register', unauthorisedInDemo, async (req, res, next) => {
     try {
+        if (req.body.password && !validatePassword(req.body.password)) {
+            next(new Error('Password does not meet requirements'));
```

Comment:
Better to use `ParameterError` than Error ? `ParameterError` returns a 400 code, while Error I think is 500 .